### ISSUE TITLE
update.sh seems to be borked

### ISSUE THREAD

**Issue by technicalpyro** [pi-hole/pi-hole#1576]
**In raising this issue, I confirm the following (please check boxes, eg [X]) Failure to fill the template will close your issue:**

- [x] I have read and understood the [contributors guide](https://github.com/pi-hole/pi-hole/blob/master/CONTRIBUTING.md).
- [x] The issue I am reporting can be *replicated*
- [x] The issue I am reporting isn't a duplicate (see [FAQs](https://github.com/pi-hole/pi-hole/wiki/FAQs), [closed issues](https://github.com/pi-hole/pi-hole/issues?utf8=%E2%9C%93&q=is%3Aissue%20is%3Aclosed%20), and [open issues](https://github.com/pi-hole/pi-hole/issues)).

**How familiar are you with the codebase?:**

_{5}_

---
**[BUG REPORT | OTHER]:**

Please [submit your feature request here](https://discourse.pi-hole.net/c/feature-requests), so it is votable by the community.  It's also easier for us to track.

**[BUG | ISSUE] Expected Behaviour:**
pihole update  when `pihole -up` is used

**[BUG | ISSUE] Actual Behaviour:**
```
pi@raspberrypi:~ $ pihole -up
  [i] Checking for updates...
/opt/pihole/update.sh: line 46: git rev-parse @{0}: command not found
/opt/pihole/update.sh: line 55: git rev-parse @{upstream}: command not found
        Error: Local revision could not be obtained, ask Pi-hole support.
        Additional debugging output:
On branch development
Your branch is behind 'origin/development' by 3 commits, and can be fast-forwarded.
  (use "git pull" to update your local branch)
nothing to commit, working directory clean
```

**[BUG | ISSUE] Steps to reproduce:**

-run the dev branch 
-try an update use git head -2 to roll back from current up to date if need be should you be fully updated
-
-

**(Optional) Debug token generated by `pihole -d`:**

`qk7s75z58l`

_This template was created based on the work of [`udemy-dl`](https://github.com/nishad/udemy-dl/blob/master/LICENSE)._

**Comment by technicalpyro** [comment]
should anyone else encounter this now that the fix has been pushed use the following steps to resolve it
```
cd /etc/.pihole/
sudo git pull
pihole -r
```

**Comment by jacobsalmela** [comment]
Resolved in v3.2 (#1577)
